paymentsheet/api/paymentsheet.api

```diff
@@ -1920,6 +1920,13 @@ public final class com/stripe/android/paymentsheet/ui/ComposableSingletons$EditP
 	public final fun getLambda-2$paymentsheet_release ()Lkotlin/jvm/functions/Function2;
 }
 
+public final class com/stripe/android/paymentsheet/ui/ComposableSingletons$NewPaymentMethodVerticalLayoutUIKt {
```

Comment:
Not really sure why this was generated now, but seems ok given that there are some others and this looks like generated code?